libproto/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
-protobuf = { version = "2.8.1", features = ["with-bytes"]}
+protobuf = { version = "=2.8.1", features = ["with-bytes"]}
```

Comment:
Does it really need to fix the protobuf version?

Reply:
> due to protobuf plguin's version is 2.8.1, need to specify protobuf version to avoid version checking error
yeah